emag-tech-labs/annotation-cache-bundle/5.0/config/packages/emag_annotation_cache.yaml

```diff
@@ -0,0 +1,3 @@
+annotation_cache:
+    provider: {}
```

Comment:
I see that this bundle does not work without a provider. Let's add `cache.app` here as a default.

Reply:
Added 'cache.app' as the default value